src/properties.cpp

```diff
@@ -4,7 +4,7 @@
 #include <morphio/errorMessages.h>
 #include <morphio/properties.h>
 
-bool verbose = false;
+constexpr bool VERBOSE = false;
```

Comment:
I think this is from earlier times, is it still needed?

Reply:
It should be removed. But I want to replace it with a helper comparator function that acts the same way as operator== except it display info to the user about what differs.